CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - ``tt restart`` confirmation prompt. ``-y`` option is added to accept restart without prompting.
 - ``tt pack`` will generate systemd unit for rpm and deb packages.
+- Support for all TT_CFG_VAR for old tarantools
```

Comment:
Set a dot at the end of the sentence.
I think it would be nice to describe in more detail what has been added and for which versions of tarantool.

Reply:
Changed